libraries/standard/http/test/Makefile

```diff
@@ -18,13 +18,16 @@ INCLUDE = ../include
 
 # functions to dump into separate source files
 # Values based on test source file names will be dynamically added.
-#FUNCTIONS = a_private_func
+FUNCTIONS = _sendHttpHeaders _sendHttpBody _receiveHttpResponse
 
 # Changes to the above variables should remain above this include.
 include Mock4thewin.mk
 
 # additional dependencies for all tests
-$(TESTS): common.h
+$(TESTS): ../../../../platform/posix/iot_clock_posix.o ../../../../platform/posix/iot_logging.o
```

Comment:
No external dependencies should exist for unit tests! The log calls should be mocked out!

Reply:
Though I understand that debugging would be easier in presence of logs

Reply:
I want this for debugging